lib/polymer/iron.dart

```diff
@@ -249,18 +187,36 @@ class CoreElement extends WebElement {
 
     return _eventStreams[eventName];
   }
+
+  void add(dynamic child) {
+    if (child is WebElement) {
+      child = child.element;
+    } else if (child is Element) {
+      child = child;
+    } else {
+      throw new ArgumentError('child must be a WebElement or an Element');
+    }
+    //Polymer.dom(this).appendChild(child);
+    context["Polymer"].callMethod("dom", [new JsObject.fromBrowserObject(element)])
+    .callMethod("appendChild", [new JsObject.fromBrowserObject(child)]);
+  }
+
+  dynamic selectorAll(String selector) {
+    //Polymer.dom(this).childNodes;
+    return context["Polymer"].callMethod("dom", [new JsObject.fromBrowserObject(element)]).callMethod("querySelectorAll", [selector]);
+  }
 }
 
 class Transition {
-  static void coreTransitionCenter(CoreElement element) =>
+  static void coreTransitionCenter(PolymerElement element) =>
       element.setAttribute('transition', 'core-transition-center');
 }
 
 class Transitions {
-  static void slideFromRight(CoreElement element) =>
+  static void slideFromRight(PolymerElement element) =>
```

Comment:
I'm wondering if the old transitions map to the new `neon-animation`s (https://elements.polymer-project.org/elements/neon-animation?active=slide-from-right-animation).